docs/start/getting-started/fragments/ios/generate-model.md

```diff
@@ -26,27 +26,9 @@ With the basic setup complete, next you will model the data your application wil
     - **priority** an optional enumeration type field that indicates the importance of a Todo item; the value of priority can be only *LOW*, *NORMAL*, or *HIGH*
     - **description** an optional string field that holds more information about a Todo item
 
-1. Next, generate the classes for these models. **Update the amplifytools.xcconfig file** by changing `modelgen=false` to:
+1. Next, generate the classes for these models and add then to your Xcode project. **Run the command**:
   ```bash
-  modelgen=true
+  amplify codegen models && amplify-app
```

Comment:
I don't recall ever having to run `amplify-app` before, but based on the next instruction of "generated models have been added to your project", i'm guessing that it auto-magically adds the files to your project?

Maybe we should separate `amplify codegen models` and `amplify-app`?  there may be people who just want to add the files manually to their project?

Reply:
That's how the DataStore getting started works, but with the goal of "minimum effort required" the tutorial just combines then otherwise we need to start forking option 1 and 2 like the DataStore docs which can get messy.

As for *"I don't recall ever having to run amplify-app before"*, it's because Amplify/Tools was doing that for you.